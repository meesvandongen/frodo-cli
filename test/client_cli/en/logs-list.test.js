import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);
const CMD = 'frodo logs list --help';
const { stdout } = await exec(CMD);

test("CLI help interface for 'logs list' should be expected english", async () => {
  expect(stdout).toMatchSnapshot();
});
