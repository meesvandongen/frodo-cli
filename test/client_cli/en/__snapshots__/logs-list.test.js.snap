// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'logs list' should be expected english 1`] = `
"Usage: frodo logs list [options] <host> [user] [password]

List available ID Cloud log sources.

Arguments:
  host            Access Management base URL, e.g.:
                  https://cdk.iam.example.com/am. To use a connection profile,
                  just specify a unique substring.
  user            Username to login with. Must be an admin user with
                  appropriate rights to manage authentication journeys/trees.
  password        Password.

Options:
  -k, --insecure  Allow insecure connections when using SSL/TLS. Has no effect
                  when using a network proxy for https
                  (HTTPS_PROXY=http://<host>:<port>), in that case the proxy
                  must provide this capability. (default: Don't allow insecure
                  connections)
  --verbose       Verbose output during command execution. If specified, may or
                  may not produce additional output.
  --debug         Debug output during command execution. If specified, may or
                  may not produce additional output helpful for
                  troubleshooting.
  --curlirize     Output all network calls in curl format.
  -h, --help      Help
"
`;
